cypress/support/step_definitions/find/i_check_the_string_checkbox_for_string.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+And("I {string} the {string} checkbox for {string}", (checkOrUncheck, checkButton, group) => {
```

Comment:
Probably a definition I created by maybe worth changing the `And` to something else 😁